### PR TITLE
Add elm section to README.md

### PR DESCRIPTION
**Pull request by VladimirLogachev**

### CODE REVIEW

**Comment by peaceiris** [comment]
Thank you for your contribution @VladimirLogachev and sorry for my late reply.

File: README.md [hunk 1]
```diff
@@ -785,6 +786,48 @@ jobs:
...
+
+      - name: Move files
+        run: |
+          mkdir ./public
+          mv ./index.html ./public/
+        # If you have non-minimal setup with some assets and separate html/js files,
+        # provide --output=<output-file> option for `elm make` and remove this step
+
+      - name: Deploy
+        uses: peaceiris/actions-gh-pages@v2.7.0-1
```
Comment (peaceiris) [inline README.md]: Please pin `v2`

File: README.md [hunk 2]
```diff
@@ -785,6 +786,48 @@ jobs:
         PUBLISH_DIR: ./build/web
 ```
 
+### ⭐️ Elm
+
+An exapmle workflow for [Elm] with [justgook/setup-elm].
+
+[Elm]: https://elm-lang.org/docs
```
Comment (peaceiris) [inline README.md]: How about `https://elm-lang.org`?

Comment (VladimirLogachev) [reply README.md]: changed to shorter url

File: README.md (hunk 1 above)
Comment (VladimirLogachev) [reply README.md]: ok, changed it

**Comment by peaceiris** [comment]
LGTM! Thank you!

**Pull request merged**